schema.sql

```diff
@@ -1,5 +1,9 @@
 /* Database schema to keep the structure of entire database. */
-
-CREATE TABLE animals (
-    name varchar(100)
-);
+CREATE TABLE animals(
+	id INT GENERATED BY DEFAULT AS IDENTITY, 
+	name VARCHAR(20), 
+	date_of_birth DATE, 
+	escape_attempts INT, 
+	neutered BIT, 
```

Comment:
- [ ] OPTIONAL: You can use BIT data type, another alternative would be `boolean` so when inserting data you do not get confused thinking 1 is for integer or BIT 😉